README.md

```diff
@@ -1,33 +1,64 @@
-# Hackers at Cambridge Website
-[![Build Status](https://travis-ci.org/hackersatcambridge/hac-website.svg?branch=master)](https://travis-ci.org/hackersatcambridge/hac-website)
+# Hackers at Cambridge Website [![Build Status](https://travis-ci.org/hackersatcambridge/hac-website.svg?branch=master)](https://travis-ci.org/hackersatcambridge/hac-website)
+> The public website of [Hackers at Cambridge]()
+
+This website is built using the Swift web framework [Kitura](https://github.com/IBM-Swift/Kitura). This allows us to build the site fast and reliably and to learn about and contribute to server-side Swift in the process. 
 
-This website is written using the Swift web framework [Kitura](https://github.com/IBM-Swift/Kitura) to allow us the flexibility to do what we please with the website and to allow us to try out Swift as an option for server-side programming.
 
 ## Installing and running the website
 
-We use Docker to make this quick and painless for you, [make sure you have it installed](https://docs.docker.com/engine/installation/).
+**Docker**
```

Comment:
These sections are good, can we make them subheadings instead of bold paragraphs?